docs/reference/commandline/index.md

```diff
@@ -48,6 +48,7 @@ read the [`dockerd`](dockerd.md) reference page.
 | [rmi](rmi.md) | Remove one or more images                                    |
 | [save](save.md) | Save images to a tar archive                               |
 | [tag](tag.md) | Tag an image into a repository                               |
+| [image prune](image_prune.md) | Remove unused images                         |
```

Comment:
perhaps we should try to keep this in alphabetical order (although the new `docker image load`, etc. , subcommands haven't been added to the docs yet.

Can you move `image prune` above `rmi` ? (same for the other changes below